config/initializers/sidekiq.rb

```diff
@@ -100,7 +100,7 @@
   end
 end
 
-Sidekiq.logger.level = ::Logger.const_get(ENV.fetch('RAILS_LOG_LEVEL', 'info').upcase.to_s)
+Sidekiq.default_configuration.logger.level = Logger.const_get(ENV.fetch('RAILS_LOG_LEVEL', 'info').upcase.to_s)
```

Comment:
Should we be touching `default_configuration` or doing that in the `Sidekiq.configure_server` (and `Sidekiq.configure_client`?) blocks?

Reply:
I'm not sure -- I probably grabbed this from an upgrade guide somewhere. I think my mental model was that if we are setting the same value in both modes this makes sense. Code-wise, they are all public methods, and both `configure_*` methods are wrapping the same memoized config object from `default_configuration`.

Looking more at it, all the examples of setting config values do use the `configure_` method blocks, all the usage of `default_configuration` are using it as a getter and not a setter (except to override in test envs), and in theory if we ever wanted diff log levels in client vs server mode that would be the way to do it.

I'm fine with whatever you prefer here.